src/Server/RestRouterHandlers/RestRouterHandler.h

```diff
@@ -117,6 +117,12 @@ class RestRouterHandler : private boost::noncopyable
         return jsonErrorResponse("Internal server error", error_code);
     }
 
+    String processQuery(
+        const String & query, const std::function<void(Block &&)> & callback = [](Block &&) {}) const;
+
+    String processQuery(
+        const String & query, Poco::JSON::Object & resp, const std::function<void(Block &&)> & callback = [](Block &&) {}) const;
```

Comment:
do we need give a default value to `callback` ?

Reply:
fixed